client/Map.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import L from 'leaflet';
+import './Map.css';
+
+const RENDERER = L.canvas({ padding: 0.5 });
+
+export default class LeafletWrapper extends Component {
```

Comment:
I recommend changing the component name to match how it is used at a higher level in `MapContainer`. 
I'm all for having something like a  `LeafletWrapper`, but that would work better as a separate utility class. Potentially easier to maintain and test that way.

```suggestion
export default class Map extends Component {
```